webapp/player-project/public/locales/de/translation.json

```diff
@@ -0,0 +1,11 @@
+{
+  "notifications": {
```

Comment:
naming: I would go with a shorter, not pluralized name such as `notif`, but really a nitpick.